dice.conf

```diff
@@ -0,0 +1,41 @@
+################################## GENERAL #####################################
```

Comment:
Can you please mention once how're we using these config vars and values?
Few of them I don't see being utilised in code, like `daemonize, bind, save, dbfilename, appendonly`?

For now we can just have `port, requirepass` as mentioned in issue and can extend later.

Reply:
other values are useful in upcoming release, consider this as patch release coz remaining option may become dependency for many commands so I would like to wait for further guidance and approvals, thank you for raising your valuable concerns